scripts/complete_liquidity_provision.js

```diff
@@ -79,9 +86,13 @@ module.exports = async (callback) => {
     const investmentTargetToken = toErc20Units(argv.investmentTargetToken, targetTokenDecimals)
     const investmentStableToken = toErc20Units(argv.investmentStableToken, stableTokenDecimals)
 
-    assert(argv.fleetSize % 2 == 0, "Fleet size must be a even number for easy deployment script")
+    if (argv.brackets.length > 0) {
+      // Override fleetSize with num brackets when provided.
+      argv.fleetSize = argv.brackets.length
+    }
+    assert(argv.fleetSize % 2 === 0, "Fleet size must be a even number for easy deployment script")
```

Comment:
It seems like fleetSize makes only sense to specify if you don't specify `--brackets`. Should we move all fleet size checks (<= 23, event etc.) into the if branch that actually uses this value (the one that calls deployFleetOfSafes)

Reply:
Federico has good examples of checking the argv arguments in the withdraw script. Especially, he has several checks that check that either the one or the other parameter is set. You can take a look over there...

Reply:
I have explicitly set `fleetSize`equal to the bracket array length so the equivalent checks can be performed without requiring any other changes. This was the least invasive and still does the same thing. I didn't really want to get involved with the business of putting a bunch of logical constraints on the command line arguments.

Reply:
> checking the argv arguments in the withdraw script. 

I will take a look, but I don't exactly feel the need to impose such complex logic unless someone would really like it. Please let me know if this is a suggestion, or a must-have.

Reply:
With the current usage of the script it makes sense to have both arguments set, then you'd also check whether the length of `argv.brackets` coincides with `argv.fleetSize` (notes on usage [here](https://github.com/gnosis/dex-liquidity-provision/issues/193#issuecomment-624198288)).

Reply:
So instead of simply resetting the value to `brackets.length` I should just assert that the length matches the `fleetSize`. I can do this and it would make thing a bit cleaner here, but also seems superfluous. 

I looked at the link you provided, but not exactly sure what relevance it has in this discussion.

Overall, I would be happy to go with the suggestion that `fleetSize` agree with `brackets.length`. It is a simple change, it makes the script more safe and does not impeded on any existing code or impose unnecessarily complex business logic. Unless otherwise stated, I will implement this option.

Reply:
Actually, I don't really see the difference apart from inconvenience from the usability perspective from doing

```js
    if (argv.brackets.length > 0) {
      // Override fleetSize with num brackets when provided.
      argv.fleetSize = argv.brackets.length
    }
```

over

```
    if (argv.brackets.length > 0) {
      // Override fleetSize with num brackets when provided.
      assert.equal(argv.fleetSize, argv.brackets.length, "Please ensure fleetSize == number of brackets")
    }
```